lib/indexable_content_checker.rb

```diff
@@ -0,0 +1,71 @@
+require 'gds_api/content_store'
```

Comment:
As discussed, can we move this class inside the rake task so that it is clearer they will both be deleted once we have the data. :)